### PR TITLE
[users] Move user provisioning to atlantis workflow

### PR DESCRIPTION
**Pull request by osterman**
## what
* Update to use atlantis
* Deprecate terragrunt workflow

## why
* Support operations by pull request

### CODE REVIEW

File: conf/users/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+
+## Adding a New User
+
+To add new users, add them to this directory.
+
+Add new users by adding a `.tf` file in this directory for their configuration.
+
+For a user with an email of `user@example.co`, their configuratino would look like this:
```
Comment (Nuru) [inline conf/users/README.md]:
```suggestion
For a user with an email of `user@example.co`, their configuration would look like this:
```

File: conf/users/.envrc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Import the remote module
+export TF_CLI_INIT_FROM_MODULE="git::https://github.com/cloudposse/terraform-root-modules.git//aws/users?ref=tags/0.53.3"
```
Comment (Nuru) [inline conf/users/.envrc]: Might want to wait until we undo default EFS provisioning.

File: conf/users/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+## Temporary Login Credentials
+
+After provisioning the user, their base64 keybase encrypted password will be output by the module. 
+
+To retrieve a user's base64 encrypted password decrypt command, run the following command:
+
+```
+terraform output user@example.co
+```
```
Comment (Nuru) [inline conf/users/README.md]: Is the generated password the same with each `terraform plan`? I thought it was random.

File: conf/users/terraform.tfvars [hunk 4]
```diff
@@ -1,17 +1,2 @@
...
-      env_vars = {
-        TF_VAR_smtp_username = "${get_env("SMTP_USERNAME", "")}"            
-        TF_VAR_smtp_password = "${get_env("SMTP_PASSWORD", "")}" 
-        TF_VAR_smtp_host = "${get_env("SMTP_HOST", "smtp.maigun.org")}"            
-        TF_VAR_smtp_port = "${get_env("SMTP_PORT", "587")}"            
-      }
-    }
-  }
-}
+smtp_host = "smtp.maigun.org"
```
Comment (Nuru) [inline conf/users/terraform.tfvars]:
```suggestion
smtp_host = "smtp.mailgun.org"
```

File: conf/users/README.md (hunk 3 above)
Comment (osterman) [reply conf/users/README.md]: Yes, it will be stable. Also, the password is generated by AWS and not terraform.

File: conf/users/README.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+**IMPORTANT** This assumes you've started the shell and run `assume-role` to login. Run `make init` to setup the remote state.
+
+## User Account Setup
+
+The user will need to then setup their AWS account. They will do this by going to the [login URL](../README.md) for the "root" account.
+
+In order to login, they will need to enter their email address followed by their temporary password which was obtained by running the decrypt command from the terraform output. 
+
+After setting up their password, the user will need to [attach an MFA device](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_mfa_enable_virtual.html) to their account or ask for from from their friendly sysadmin. Then they will need to log out and log back in again using their MFA device. Using this new login session, then navigate to the [IAM menu and obtain the AWS credentials](https://aws.amazon.com/blogs/security/wheres-my-secret-access-key/) (*AWS Access Key ID* and *AWS Secret Access Key*) for their account. These will be needed when setting up their shell with `aws-vault`.
```
Comment (aknysh) [inline conf/users/README.md]:
```suggestion
After setting up their password, the user will need to [attach an MFA device](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_mfa_enable_virtual.html) to their account or ask for from their friendly sysadmin. Then they will need to log out and log back in again using their MFA device. Using this new login session, then navigate to the [IAM menu and obtain the AWS credentials](https://aws.amazon.com/blogs/security/wheres-my-secret-access-key/) (*AWS Access Key ID* and *AWS Secret Access Key*) for their account. These will be needed when setting up their shell with `aws-vault`.
```

**Comment by osterman** [comment]
atlantis apply -p users

**Pull request merged**